src/main/java/gov/nih/nci/evs/api/service/SparqlQueryManagerServiceImpl.java

```diff
@@ -2442,6 +2451,7 @@ public List<Concept> getAllConceptsWithCode(final Terminology terminology) throw
     try {
       res = restUtils.runSPARQL(queryPrefix + query, getQueryURL());
     } catch (final Exception e) {
+      auditError(e, "getAllConceptsWithCode");
       e.printStackTrace();
```

Comment:
This is one of those situations - just remove the try complete.  it's not doing anything.  The error will bubble up.